flycheck.el

```diff
@@ -8950,6 +8951,35 @@ See URL `https://docs.python.org/3.4/library/py_compile.html'."
           line ", " column ", " (one-or-more not-newline) line-end))
   :modes python-mode)
 
+(flycheck-def-config-file-var flycheck-python-mypy-ini python-mypy
+                              "mypy.ini"
+  :safe #'stringp)
+
+(flycheck-def-option-var flycheck-python-mypy-disable-incremental nil python-mypy
+  "Disable incremental type checking in mypy.
+
+When non-nil, mypy will run more slowly, but it will not create
+.mypy_cache files."
+  :safe #'booleanp
+  :type 'boolean
+  :package-version '(flycheck . "0.16"))
```

Comment:
The version number should be `32`, not `0.16`.

Reply:
Fixed.